src/pages/Reviews/SingleReview.jsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from "react";
+import { useLoaderData, useNavigate } from "react-router-dom";
+import reqURLs from "../../api";
+import { NotFound } from "../NotFound/NotFound";
+import { Helmet, HelmetProvider } from "react-helmet-async";
+
+export default function SingleReview() {
+  const navigate = useNavigate();
+  const [user, setUser] = useState("");
+
+  const { review } = useLoaderData();
+
+  useEffect(() => {
+    const getUser = async () => {

```

Comment:
This looks like an API function so should really be extracted into the seperate API file, as you may need to getUsers in other areas of the app this would prevent duplicate logic.